docs/community/release_notes/release_note_0.3.0.md

```diff
@@ -0,0 +1,369 @@
+---
+hide:
+  - navigation
+
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+# Apache Celeborn(Incubating) 0.3.0 Release Notes
+
+## Highlight
+
+- Initial support for Flink 1.14 1.15 1.17
+
+- Initial support for Spark 3.4
+
+- Initial support for MapPartition on shuffle write and read and support for credit-based shuffle read to improve performance
+
+- Compatibility support for version 0.2.x and 0.3.0
+
+- Add support for batch revive RPCs in clients to avoid too many requests
+
+- Enhanced worker exclusion mechanism
+
+- Optimization of memory usage by pooling flusher's CompositeByteBuf and PushTask queues 
+
+- Enhanced rolling upgrading and graceful shutdown
+
+- More bug fixes and usability improvements of tiered storage
+
+- Performance improvements and bug fixs of native Spark columnar shuffle
+
+- More bug fixes and usability improvements of K8S deployment
+
+- More code reflectors and code refines
+
+- More configuration default values are changed and new confs are added
+
+### Spark/Flink
+- FLINK: Implement Flink 1.14 plugin [CELEBORN-235]
```

Comment:
How about moving the ticket ahead? (No strong opinion)

```suggestion
- [CELEBORN-235] FLINK: Implement Flink 1.14 plugin
```

Reply:
ok. i move them later.